app/controllers/provider/admin/account/authentication_providers_controller.rb

```diff
@@ -41,14 +52,16 @@ def update
     if @authentication_provider.update(authentication_provider_params)
       redirect_to provider_admin_account_authentication_provider_path(@authentication_provider), notice: 'SSO integration updated'
     else
-      flash.now[:error] = 'SSO integration could not be updated'
+      flash.now[:error] = t('.error')
       render :edit
     end
   end
 
   def destroy
-    authentication_provider.destroy
-    redirect_to provider_admin_account_authentication_providers_path, notice: 'SSO integration deleted'
+    return unless authentication_provider.destroy
```

Comment:
If `authentication_provider.destroy` returns `false`, then we return without any error? IS that expected?

Reply:
Might have been rubocop, I'll revert it.